includes/class-simple-local-avatars.php

```diff
@@ -251,6 +257,13 @@ public function admin_init() {
 				'desc' => __( 'Only allow users with file upload capabilities to upload local avatars (Authors and above)', 'simple-local-avatars' ),
 			)
 		);
+		add_settings_field(
+			'simple-local-avatars-migration',
+			__( 'Migrations', 'simple-local-avatars' ),
```

Comment:
I'm not looking at this in the context of the full screen at the moment but I wonder if this might need to be a more descriptive section title like "Migrate Other Local Avatars" or something like that.

Reply:
Done